torchvision/csrc/io/image/cpu/decode_jpeg.cpp

```diff
@@ -67,6 +67,61 @@ static void torch_jpeg_set_source_mgr(
   src->pub.next_input_byte = src->data;
 }
 
+inline void _convert_pixel_cmyk_to_rgb(
+    bool has_adobe_marker,
+    int c,
+    int m,
+    int y,
+    int k,
```

Comment:
would it help to mark them as `const`?

Reply:
It is an inline function and we rarely put const to basic types like int.